frontend/components/product-list/sections/FilterableProductsSection/RefinementSingleSelect.tsx

```diff
@@ -68,66 +64,78 @@ export function RefinementMenu({
    );
 }
 
-type MenuItemProps = {
-   label: string;
-   value: string;
-   count: number;
-   isRefined: boolean;
+type SingleSelectItemProps = {
+   item: RefinementListRenderState['items'][0];
    attribute: string;
    refine: RefinementListRenderState['refine'];
    onClose?: () => void;
 };
 
-const MenuItem = React.memo(function RefinementListItem({
-   label,
-   count,
-   value,
-   isRefined,
+const SingleSelectItem = React.memo(function SingleSelectItem({
+   item,
    attribute,
    refine,
    onClose,
-}: MenuItemProps) {
+}: SingleSelectItemProps) {
+   const [isRefined, setIsRefined] = useDecoupledState(item.isRefined);
    const { refine: clearRefinements } = useClearRefinements({
       includedAttributes: [attribute],
    });
    const { createURL } = useCurrentRefinements();
-   const url = new URL(
-      createURL({
-         attribute,
-         type: 'disjunctive',
-         value,
-         label,
-      })
+
+   const TitleText = (
+      <Text
+         role="option"
+         data-value={item.value}
+         as={attribute === 'facet_tags.Item Type' ? 'a' : 'button'}
+         onClick={(event) => {
+            event.preventDefault();
+            setIsRefined((current) => !current);
+            clearRefinements();
+            refine(item.value);
+            onClose?.();
+         }}
+         _hover={{
+            textDecoration: 'underline',
+         }}
+      >
+         {item.label}
+      </Text>
    );
-   // The url created by InstantSearch doesn't have the correct item type slug.
-   const path = url.pathname.split('/').filter((part) => part !== '');
-   const itemTypeHandle = encodeDeviceItemType(value);
-   const href = `${url.origin}/${path[0]}/${path[1]}/${itemTypeHandle}${url.search}`;
+
+   let RefinementTitle;
+   if (attribute === 'facet_tags.Item Type') {
+      const url = new URL(
+         createURL({
+            attribute,
+            type: 'disjunctive',
+            value: item.value,
+            label: item.label,
+         })
+      );
+      // The url created by InstantSearch doesn't have the correct item type slug.
+      const path = url.pathname.split('/').filter((part) => part !== '');
+      const itemTypeHandle = encodeDeviceItemType(item.value);
+      const href = `${url.origin}/${path[0]}/${path[1]}/${itemTypeHandle}${url.search}`;
```

Comment:
We got rid of the origin in another pull since relative urls will work fine here and it prevents a mismatch between server and client rendering.
```suggestion
      const href = `/${path[0]}/${path[1]}/${itemTypeHandle}${url.search}`;
```